jest.config.js

```diff
@@ -11,7 +12,7 @@ module.exports = {
   testMatch: ['**/*.test.ts'],
   transform: { '\\.ts$': 'ts-jest' },
   coverageReporters: ['lcov', 'text-summary'],
-  collectCoverage: !!process.env.CI,
+  collectCoverage,
```

Comment:
Manually specify which version to collect coverage on CI would be better I think:

```suggestion
  collectCoverage: !!process.env.ENABLE_COVERAGE,
```

```diff
 # .travis.yml
-yarn run test
+ENABLE_COVERAGE=1 yarn run test
```